administrator/language/en-GB/plg_task_updatenotification.ini

```
@@ -4,7 +4,8 @@
 ; Note : All ini files need to be saved as UTF-8
 
 PLG_TASK_UPDATENOTIFICATION="Task - Joomla! Update Notification"
-PLG_TASK_UPDATENOTIFICATION_EMAIL_DESC="A comma separated list of the email addresses which will receive the update notification emails. The addresses in the list MUST belong to existing users of your site who have the Super User privilege. If none of the listed emails belongs to Super Users, or if it's left blank, all Super Users of this site will receive the update notification email."
+ PLG_TASK_UPDATENOTIFICATION_END="Update Notification end"
+ PLG_TASK_UPDATENOTIFICATION_EMAIL_DESC="A comma separated list of the email addresses which will receive the update notification emails. The addresses in the list MUST belong to existing users of your site who have the Super User privilege. If none of the listed emails belongs to Super Users, or if it's left blank, all Super Users of this site will receive the update notification email."
```

```suggestion
PLG_TASK_UPDATENOTIFICATION_END="Update Notification end"
PLG_TASK_UPDATENOTIFICATION_EMAIL_DESC="A comma separated list of the email addresses which will receive the update notification emails. The addresses in the list MUST belong to existing users of your site who have the Super User privilege. If none of the listed emails belongs to Super Users, or if it's left blank, all Super Users of this site will receive the update notification email."
```
Revert space at beginning of line.